sdk/security/keyvault/azadmin/rbac/client_test.go

```diff
@@ -0,0 +1,240 @@
+//go:build go1.18
+// +build go1.18
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package rbac_test
+
+import (
+	"context"
+	"math/rand"
+	"testing"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/to"
+	"github.com/Azure/azure-sdk-for-go/sdk/internal/recording"
+	"github.com/Azure/azure-sdk-for-go/sdk/security/keyvault/azadmin/rbac"
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/require"
+)
+
+func TestRoleDefinition(t *testing.T) {
+	client := startAccessControlTest(t)
+
+	var name, roleName string
+	if recording.GetRecordMode() == recording.PlaybackMode || recording.GetRecordMode() == recording.RecordingMode {
+		name, roleName = "bedc5fb2-8738-40a4-8b20-cedfb43a1922", "c023eb03-4e31-464c-84f7-001b7f23bd13"
+	} else {
+		name, roleName = uuid.New().String(), uuid.New().String()
```

Comment:
I take it you need the same UUIDs in playback and recording but the specific values don't matter? It would be tidier to use a [variable](https://github.com/Azure/azure-sdk-tools/tree/main/tools/test-proxy/Azure.Sdk.Tools.TestProxy#storing-variables) (haven't tried myself) or configure a sanitizer to replace the recorded value with a static fake value.

Reply:
Will fix in next release